linera-indexer/lib/src/rocks_db.rs

```diff
@@ -45,8 +45,11 @@ impl RocksDbRunner {
         };
         let path_buf = config.client.storage.as_path().to_path_buf();
         let path_with_guard = PathWithGuard::new(path_buf);
+        // The tests are run in single threaded mode
+        let block_in_place = false;
```

Comment:
More importantly, it's a safe default value.

Reply:
Ok, added.